infrastructure/aws/variables.tf

```diff
@@ -15,19 +15,19 @@ variable "public_domain" {
 
 variable "admin_1_public_key" {
   type    = string
-  default = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDEeeH3pe0IKz2OE4jVxYKycwhz4Vywkcima+vk0XF/oqMSc+uR5Dqw7/y3/4yysch3+/adeIfY6Vb11akZvCcS22/1O9EtpHupB2noRLgcOCBhRwBR+EGgUrhcMWEzJ8yNaqpC2Z8GvhJcO7F3rlktISVh/GOXI/91KcapU3v9I+ASELbCaQm9mdNQBvf2vyRzWhRW18bgi43r9RmwPNI3B/xJIkYFFa7OnUBGTvi1PW9X8ZQkDGwTT0b4fAKP4tl6UaXOkF3vpIT1gmyL4jiZccDKFJkKdTxtyt67lON22MWKmjrCs8nASX4syyefAbA3tHREYdPeZJTtx95r3Puv"
+  default = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC6W5QdfbR+2Lnfsp39i/jgjv1Ot7B71rPDZLajUHFMmo+j6YBVffm8K9gLS9omKlAl4LciA/Oy+LVesV5apeMRgYKsWIpPo6dh9BCCi+ip6vyvhUzvJjNBUMY6wIukBuAxpNEtF9DT/a1lCgDbAMnz9wVyqPDeoEPbWd9ufVJJg5+K96yRMKkjNio1Me+9o/ZbgGC85rCzdTq88wR1ilp7yhvCp5a+VpIdNlU2TlVZEq4R2CYu0xdKiaLy92ABo7Z3btetCRqruXfvhD7slkcDr88IqYCQptD2jhQu2F5Y++hU9CiOkeApaFrnGyKYtybRqI0/ISHYwFDFMN/IHvOzZV3NJpvmF3twIe7Y9437gW6Nqxkdr5EVDbICX9KdZKMIBU8ZmsFq26AP/cWAezo9xoNiqoWtnCeXwO1vCG2azdMuyAJ8GfiypWbYZoURVaKOmb44/yIUx3d8MlX/wqbGGrA4CpCFBwydLj5MNPVA19Y4M41hrxLQEaMudMJAvXE="
 }
 
 variable "static_content_bucket_name" {
   type    = string
-  default = "nedluzim-statu-static-deployment"
+  default = "nedluzim-statu-static-deployment-iprp"
 }
 
 variable "https_certificate" {
   type    = string
-  default = "arn:aws:acm:us-east-1:313370994665:certificate/1eb30f1c-0b56-4bae-8530-ace418b8637a"
+  default = "arn:aws:acm:us-east-1:734731567119:certificate/79422c20-6f85-41ed-ad52-e0da62d4a57f"
```

Comment:
@miiila - jen koukam ze ten certifikat je v us-east-1 a ne v eu-central-1. Uplne nevim co vsechno to muze zpusobit, ale zvazil bych ho dat do eu-central-1

Reply:
Tohle je bohuzel vlastnost AWS. Certifikaty, ktere jsou na EDGE lokacich (jako CloudFront) musi byt v us-east-1 zone.